supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -90,7 +90,7 @@
 [discrete]
 [[gcc-command]]
 ==== image:images/yes.png[yes] gcc (noun)
-*Description*: "GCC" is an abbreviation for the "GNU Compiler Collection". GCC is a compiler system produced by the GNU Project supporting various programming languages. When referring to the command, use `gcc`, marked up appropriately.
+*Description*: _GCC_ is an abbreviation for the "GNU Compiler Collection". GCC is a compiler system produced by the GNU Project supporting various programming languages. When referring to the command, use `gcc`, marked up appropriately.
```

Comment:
Some entries surround abbreviated terms in quotation marks and others do not. See "GB" versus "Gbps".

Reply:
Updated throughout to make more consistent.